src/JwplatformClient.php

```diff
@@ -673,6 +673,37 @@ public function query_usage($site_id, $body = null, $query_params = null) {
 class _ThumbnailClient extends _ResourceClient {
 
     protected $_resource_name = "thumbnails";
+
+    public function create($site_id, $body = null, $query_params = null) {
+        return $this->_client->request(
+            "POST",
+            "/v2/sites/" . $site_id . "/" . $this->_resource_name,
+            $body,
+            [],
+            $query_params
+        );
+    }
+
+    public function get_details($site_id, $resource_id, $body = null, $query_params = null) {
```

Comment:
```suggestion
    public function get($site_id, $resource_id, $body = null, $query_params = null) {
```

Probably should just be get to align with [other methods like this](https://github.com/jwplayer/jwplatform-php/blob/6cfe654554f717a75380ef8987d90a30e25bd55e/src/JwplatformClient.php#L227) unless details has a specific meaning here.